app/packages/components/src/components/index.ts

```diff
@@ -37,5 +37,6 @@ export { default as TabOption } from "./TabOption";
 export { default as TextField } from "./TextField";
 export { default as ThemeProvider, useFont, useTheme } from "./ThemeProvider";
 export { default as Tooltip } from "./Tooltip";
+export { Toast } from "./Toast";
```

Comment:
```suggestion
export { default as Toast } from "./Toast";
```
Can we follow the existing pattern here, to keep this file consistent?